testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -1786,6 +1911,9 @@ fn end_of_tenure() {
         .reward_cycle_to_block_height(final_reward_cycle)
         - 2;
 
+    // give the system a chance to mine a Nakamoto block
+    sleep_ms(30_000);
```

Comment:
Is there anything triggering a block to be produced?

Reply:
It's waiting for the Stacks chain tip to advance.  I'll refactor to use `wait_for()`.